cli/src/wallet.rs

```diff
@@ -457,6 +488,49 @@ pub fn parse_create_address_with_seed(
     })
 }
 
+pub fn parse_find_program_address(matches: &ArgMatches<'_>) -> Result<CliCommandInfo, CliError> {
+    let program_id = resolve_derived_address_program_id(matches, "program_id").unwrap();
```

Comment:
I know that `create-address-with-seed` has the same issue, but can we return `CliError::BadParameter` here instead of unwrapping to terminate more gracefully?